docs/scripting/functions/AddSimpleModel.md

```diff
@@ -60,4 +60,4 @@ There are currently no restrictions on when you can call this function, but be a
 
 ## Related Functions
 
-- OnPlayerFinishedDownloading: Called when a player finishes downloading custom models.
+- [OnPlayerFinishedDownloading](../../scripting/callbacks/OnPlayerFinishedDownloading.md): Called when a player finishes downloading custom models.
```

Comment:
Just a note, you're already _inside_ the `scripting` directory, so you only need to `../` once and omit `scripting`

`../callbacks/OnPlayerFinishedDownloading.md`

But I'll fix all of these automatically, it doesn't really matter, it just makes the lines shorter.